src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -1689,7 +1691,6 @@ public void testEColiFuncotations() {
         arguments.add(FuncotatorArgumentDefinitions.TRANSCRIPT_SELECTION_MODE_LONG_NAME, TranscriptSelectionMode.CANONICAL.toString());
         runCommandLine(arguments);
         assertEqualVariantFiles(outputFile, E_COLI_EXPECTED_OUT);
-
     }
```

Comment:
Can you add new `Funcotator` tests to cover the fixes in this branch (the hg38 dbsnp contig names fix and the MAF tab fix)?

Reply:
I'll add one in for the MAF trailing tab fix.

I can add in a trivial test for hg38 dbsnp with the updated contig names, but I'm not sure it adds much value - I'd just be using an excerpt of that file to create funcotations.  The issue for that was really in the datasource as opposed to the code, so while adding in the test would show that the latest lifted over file works, it doesn't actually reflect the data source in the package we provide.  That is, this test would need to be updated every time we update the dbSNP data source so we have an excerpt from it here.  I think what makes more sense is to refresh the test datasource for dbSNP with the latest version - then the large scale tests can make sure that it works properly.

@droazen What do you think?

Reply:
Actually, rather than adding in a test for the trailing tab thing, I'm adding a new overload for `IntegrationTestSpec::assertEqualTextFiles` to allow for whitespace to **not** be ignored.  Were it not for this, this trailing tab issue would have been found a long time ago.